docs/guide/essentials/history-mode.md

```diff
@@ -70,6 +70,29 @@ http.createServer((req, res) => {
 
 For Node.js/Express, consider using [connect-history-api-fallback middleware](https://github.com/bripkens/connect-history-api-fallback).
 
+##### Example
+```bash
+mkdir express_project
```

Comment:
could you use `express-project` instead?

Reply:
yes, I fixed.